### ISSUE TITLE
Use npm release tagging when releasing alphas & betas

### ISSUE THREAD

**Issue by dschep** [JedWatson/react-select#2348]
Would it be possible to release the module to npm using `alpha` and `beta` tags instead of `latest` when uploading alphas and betas? This would prevent people from unintentionally updating to or installing an alpha version rather than the latest stable release.

https://docs.npmjs.com/getting-started/using-tags

**Comment by JedWatson** [comment]
I have been using the “next” tag for the alphas, although I had a mishap with the last release and it was mis-tagged for a couple of hours.

Are you still seeing an issue with the way it’s tagged? I verified after I fixed it on both npm and unpkg.

If anyone knows of a good way to ensure a branch is tagged correctly when it’s published please let me know.

**Comment by dschep** [comment]
Oh, great. Nevermind then. I made this issue because I got a pull request from @greenkeeperio for 2.0.0-alpha.3, which must've picked it up during the couple of hours that it was tagged as `latest`

Yeah, I don't know of a good way to avoid making this mistake. I wish @npm would  just do smarter automatic tagging of prereleases since they require semver which is meant to be machine parseable, so it would be pretty easy to implement, I'd think.
